mockwebserver-deprecated/src/main/kotlin/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -84,27 +78,14 @@ class MockWebServer :
   }
 
   @JvmName("-deprecated_port")
-  @Deprecated(
```

Comment:
UN-deprecating stuff from the old MockWebServer. This whole package is obsolete and these deprecation warnings aren’t benefiting anyone.